.github/workflows/integrationTests.yml

```diff
@@ -7,18 +7,18 @@ on:
 
 jobs:
 
-  test-12-0-Integration-build:
-    name: Run Tests 12.0
-    runs-on: macos-12
+  test-13-Integration-build:
+    name: Run Tests MacOS 13.0
+    runs-on: macos-13
     strategy:
       fail-fast: false
       matrix:
-        xcode: ["13.3.1", "14.0", "14.1", "14.2"]
+        xcode: ["Xcode_14.3"]
```

Comment:
Maybe several Xcode version were selected here to test using several compiler versions. Are we ok dropping those?

Reply:
Only Xcode 14.1 and later are available on the macOS 13 runners (which I had to upgrade to because of the new version of opentelemetry-swift), I can re-add 14.1 and 14.2 though if we want to make sure it continues to build there.

Reply:
👍🏼 14.2 and 14.3.1 might make sense since they use two different swift stack https://swiftversion.net/ no need for 14.1.

Reply:
Do you recommend having both 14.3 and 14.3.1?

Reply:
Just 14.2 and 14.3.1 should be enough. The later fixed a bug on 14.3 so having both isn't necessary.